ParisBNF/abb/BNFabb150.xml

```diff
@@ -28,6 +28,140 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                            </item>
+                            <item xml:id="q3" n="2">
+                              <num value="2">፪</num>
+                              <dim unit="leaf">8</dim>
+                              <locus from="10r" to="17v"/>
+                            </item>
+                            <item xml:id="q4" n="3">
+                              <num value="3">፫</num>
+                              <dim unit="leaf">8</dim>
+                              <locus from="18r" to="25v"/>
+                            </item>
+                            <item xml:id="q5" n="4">
+                              <num value="4">፬</num>
+                              <dim unit="leaf">8</dim>
+                              <locus from="26r" to="33v"/>
+                            </item>
+                            <item xml:id="q6" n="5">
+                              <num value="5">፭</num>
+                              <dim unit="leaf">8</dim>
+                              <locus from="34r" to="41v"/>
+                            </item>
+                            <item xml:id="q7" n="6">
+                              <num value="6">፮</num>
+                              <dim unit="leaf">7, half folio after 7</dim>
+                              <locus from="42r" to="48v"/>
+                            </item>
+                          </list>
+                          </collation>
+                        <condition key="deficient"/>
+                      </supportDesc>
+                      <layoutDesc>
+                        <layout columns="2" writtenLines="12 14">   
+                          <ab type="ruling">Ruling is partly visible. </ab>
+                          <ab type="pricking"> Pricking is partly visible.</ab>
+                          <ab type="ruling" cert="unknown"> The upper line is written above the ruling. </ab>
+                        </layout>
+                      </layoutDesc>
+                    </objectDesc>
+                    <handDesc>
+                      <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th-century script</seg> 
+                      </handNote>
+                    </handDesc>
+                    <additions>
+                      <list>
+                        <item xml:id="a1">
+                          <locus target="#1"/>
+                          <desc type="GuestText">An incomplete prayer against eye and head pain. <bibl>
+                            <ptr target="bm:Chaine1912Catalogue"/>
+                            <citedRange unit="page">91</citedRange>
+                            <citedRange unit="number">150</citedRange>
+                          </bibl> defined it as 'two magic formulas'.</desc>
+                          <q xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ ጸሎተ፡ በእንተ፡ ሕምም፡ አይን፡ ወርዕስ፡ መዕመን፡ ኦሪት፡ ወመድኃኒት፡ ዘበምራቅከ፡ ትከሥት፡ አይንተ፡ ዕዉራን፡</q>
+                          <note>According to <bibl>
```

Comment:
I think it is easier for the reader if the information from note is integrated into desc. Note should only be a last resort if no other element makes sense

Reply:
> I think it is easier for the reader if the information from note is integrated into desc. Note should only be a last resort if no other element makes sense
Why would I include into description something that is missing? It is an additional information from the catalogues about something that is not there anymore. I do not see how I can integrate it! But probably this note belong somewhere else?

Reply:
then leave it